include/ginkgo/core/base/executor.hpp

```diff
@@ -1602,6 +1602,11 @@ class CudaExecutor : public detail::ExecutorBase<CudaExecutor>,
      */
     cublasContext* get_cublas_handle() const { return cublas_handle_.get(); }
 
+    /**
+     * @copydoc get_cublas_handle()
+     */
+    cublasContext* get_blas_handle() const { return get_cublas_handle(); }
```

Comment:
I guess it would make sense to deprecate the other version in the near future.

Reply:
Agreed